src/repository.js

```diff
@@ -194,6 +194,7 @@ export class Repository {
       }
 
       let repository     = this.entityManager.getRepository(entityMetadata.fetch('associations', key).entity);
+
```

Comment:
ah, ähm yes, that's ~ 'newline after variable declaration' or so (var, let , const)